Sources/StreamChatTestTools/Models/ChatChannel_Mock.swift

```diff
@@ -102,4 +102,48 @@ public extension _ChatChannel {
             latestMessages: latestMessages
         )
     }
+    
+    /// Creates a new `_ChatChannel` object for  from the provided data.
+    static func mockDMChannel(
+        name: String? = nil,
+        imageURL: URL? = nil,
+        lastMessageAt: Date? = nil,
+        createdAt: Date = .init(),
+        updatedAt: Date = .init(),
+        deletedAt: Date? = nil,
+        createdBy: _ChatUser<ExtraData.User>? = nil,
+        config: ChannelConfig = .mock(),
+        isFrozen: Bool = false,
+        members: Set<_ChatChannelMember<ExtraData.User>> = [],
```

Comment:
A corner for thinking and probably nit: 

DM should be always 2 people, what happens when you set 3 members to DM Channel? 
I know this is mock and everything, but a bit more convenient from API user point would be if you would have just `member` argument and set the 1-element set in this function...

Reply:
Actually, that's not entirely true. We call it direct message but it should be rather called something like member-defined groups. They are channels implicitly identified by their members. Aka, if you can have only one channel with the given combination of members.